src/main/cli/settings.c

```diff
@@ -1332,6 +1332,8 @@ const clivalue_t valueTable[] = {
 // PG_PINIO_CONFIG
 #ifdef USE_PINIO
     { "pinio_config", VAR_UINT8 | MASTER_VALUE | MODE_ARRAY, .config.array.length = PINIO_COUNT, PG_PINIO_CONFIG, offsetof(pinioConfig_t, config) },
+    { "pinio_init", VAR_UINT8 | MASTER_VALUE | MODE_ARRAY, .config.array.length = PINIO_COUNT, PG_PINIO_CONFIG, offsetof(pinioConfig_t, init) },
```

Comment:
There are enough free bits in `pinio_config`, just use one as `init` value ...

Reply:
Yeah, I've given the bit field approach a thought for a split second, but resorted to the independent variable as overloading different bits in UI would make it error prone. Of course, there is an existing bit field config for serial, though. Thoughts?

Reply:
I'm convinced!

Reply:
I agree with @ledvinap there - it seems to be inconsistent to bit-stuff other pinio config into `pinio_config`, and then use a separate boolean for `init`.